azure-proto-network/Placeholder/SubnetData.cs

```diff
@@ -4,30 +4,21 @@
 
 namespace azure_proto_network
 {
-    /// <summary>
-    /// A class representing the subnet data model.
-    /// </summary>
+    /// <summary> A class representing the subnet data model. </summary>
     public class SubnetData : ProxyResource<Azure.ResourceManager.Network.Models.Subnet>
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="SubnetData"/> class.
-        /// </summary>
+        /// <summary> Initializes a new instance of the <see cref="SubnetData"/> class. </summary>
```

Comment:
this shouldn't be 1 line, the properties are copy paste from the existing code gen so they can be 1 line, but summary in general should be multi-line